tests/features/communities/eupl/jla.feature

```diff
@@ -257,105 +257,171 @@ Feature:
+      | Sublicence        |          |            |
+      | Compatible        | GPL-2.0+ | Apache-2.0 |
+      | None N/A          |          |            |
+      | Permissive        |          |            |
+      | GPL               |          |            |
+      | Other copyleft    |          |            |
+      | Linking freedom   |          |            |
+      | Multilingual      |          |            |
+      | For data          |          |            |
+      | For software      |          |            |
+      | Law               | GPL-2.0+ | Apache-2.0 |
+      | EU/MS law         |          |            |
+      | US law            |          |            |
+      | Licensor's law    |          |            |
+      | Other law         |          |            |
+      | Not fixed/local   |          |            |
+      | Venue fixed       |          |            |
+      | Support           | GPL-2.0+ | Apache-2.0 |
+      | Strong Community  |          | x          |
+      | Governments/EU    |          | x          |
+      | OSI approved      |          |            |
+      | FSF Free/Libre    |          |            |
+
+  @javascript
+  Scenario: Add extra licences to the comparison page.
+    Given SPDX licences:
+      | uri                              | title      | ID         |
+      | http://joinup.eu/spdx/Apache-2.0 | Apache-2.0 | Apache-2.0 |
+      | http://joinup.eu/spdx/GPL-2.0+   | GPL-2.0+   | GPL-2.0+   |
+      | http://joinup.eu/spdx/BSL-1.0    | BSL-1.0    | BSL-1.0    |
+      | http://joinup.eu/spdx/0BSD       | 0BSD       | 0BSD       |
+      | http://joinup.eu/spdx/UPL-1.0    | UPL-1.0    | UPL-1.0    |
+      | http://joinup.eu/spdx/LGPL-2.1   | LGPL-2.1   | LGPL-2.1   |
+    And licences:
+      | uri                               | title             | spdx licence | legal type                                                            |
+      | http://joinup.eu/licence/apache20 | Joinup Apache-2.0 | Apache-2.0   | Strong Community, Royalty free, Modify, Governments/EU, Use/reproduce |
+      | http://joinup.eu/licence/gpl2plus | Joinup GPL-2.0+   | GPL-2.0+     | Distribute                                                            |
+      | http://joinup.eu/licence/bsl1     | Joinup BSL-1.0    | BSL-1.0      | Distribute, Modify                                                    |
+      | http://joinup.eu/licence/0bsd     | Joinup 0BSD       | 0BSD         | Distribute, Royalty free                                              |
+      | http://joinup.eu/licence/upl1     | Joinup UPL-1.0    | UPL-1.0      | Distribute, Royalty free, Governments/EU                              |
+      | http://joinup.eu/licence/lgpl21   | Joinup LGPL-2.1   | LGPL-2.1     | Distribute, Royalty free, Governments/EU, Place warranty              |
+
+    Given I am an anonymous user
+
+    When I visit "/licence/compare/GPL-2.0+;Apache-2.0;0BSD;UPL-1.0;LGPL-2.1"
```

Comment:
I would swap `GPL-2.0+` with `LGPL-2.1` so that we can test how the "+" character is encoded by the Javascript snippet, when adding the `GPL-2.0+` licence.